src/video-display.cc

```diff
@@ -100,6 +100,7 @@ VideoLoader::~VideoLoader() {
     avcodec_close(codec_context_);
     sws_freeContext(sws_context_);
     avformat_close_input(&format_context_);
+    avcodec_free_context(&codec_context_);
```

Comment:
Given that `format_context_` is allocated before `codec_context_` and the latter somewhat depends on the former, we should probably have `codec_context_` deallocate first (classical RAII deallocation).

Reply:
To follow the same line of reasoning, I moved the sws_freeContext to be on top.